### PR TITLE
Add Trident Provisioner Disabled Statement

### PR DESCRIPTION
**Pull request by patricksimonian**

### CODE REVIEW

File: _posts/2020-10-09-trident-provisioner-disabled.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+This issue does **not impact current pvcs**. 
+
+## Impact
+
+- no new netapp pvcs will be provisionable while the Trident Provisioner is turned off
+- this will likely impact your development/promotion to production services if you are planning on provisioning **new** pvcs 
+
+## How long will provisioning be disabled?
+
+***
```
Comment (mitovskaol) [inline _posts/2020-10-09-trident-provisioner-disabled.md]:
```suggestion
We will post an update about the timeline as soon as we hear back from NetApp support that is testing a downgrade path for us.
```

**Review by mitovskaol — APPROVED** [review]

**Review by mitovskaol — APPROVED** [review]

File: _posts/2020-10-09-trident-provisioner-disabled.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "Trident Provisioner Temporarily Disabled"
+date: 2020-10-09 11:45:00 +0000
+categories: openshift update
+---
+
+As an ongoing action to yesterdays Service Outage. The Trident Provisioner Service for Openshift 3.11 is being turned off while we look to resolve the issue.
```
Comment (patricksimonian) [inline _posts/2020-10-09-trident-provisioner-disabled.md]:
```suggestion
As an ongoing action to yesterday's Service Outage, the Trident Provisioner Service for Openshift 3.11 is being turned off while we look to resolve the issue. 
```

File: _posts/2020-10-09-trident-provisioner-disabled.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+- no new netapp pvcs will be provisionable while the Trident Provisioner is turned off
+- this will likely impact your development/promotion to production services if you are planning on provisioning **new** pvcs 
+
+## How long will provisioning be disabled?
+
+We will post an update about the timeline as soon as we hear back from NetApp support that is testing a downgrade path for us.
+
+## What can you do in the mean time?
+
+- If being unable to create **new** pvcs is causing blockers for your development. As a temporary work around you can switch to ephemeral storage for services that require data. This is something that you should do for **DEV** environments only.
```
Comment (patricksimonian) [inline _posts/2020-10-09-trident-provisioner-disabled.md]:
```suggestion
- If being unable to create **new** pvcs is causing blockers for your development, as a temporary work around you can switch to ephemeral storage for services that require data. This is something that you should do for **DEV** environments only - do not plan on promoting your infrastructure changes as this is a temporary work around
```

**Review by jefkel — APPROVED** [review]
I like it.

**Pull request merged**